init.php

```diff
@@ -1,3 +1,6 @@
 <?php
+$timezone = date_default_timezone_set('Europe/Moscow');
```

Comment:
Функция возвращает bool, тут либо анализировать что вернулось, либо ничего не делать.

Reply:
```suggestion
$timezone = date_default_timezone_set('Europe/Moscow');
```
Оставляю просто date_default_timezone_set('Europe/Moscow'), а переменную убираю.